charts/prometheus-node-exporter/values.yaml

```diff
@@ -478,3 +478,6 @@ extraManifests: []
   #     name: prometheus-extra
   #   data:
   #     extra-data: "value"
+
+# Override version of app, required if image.tag is defined and does not follow semver
+version: ""
```

Comment:
Please, fix the linting error (missing new line character at the end of the file).

Reply:
Weird it only showed up now though...